CHANGELOG.md

```diff
@@ -3,9 +3,14 @@ cross-compilation issue, but will return in v0.13.0.
 
 # Main (unreleased)
 
+# 0.14.0-rc.3 (2021-04-15)
+
 - [ENHANCEMENT] Add  `headers` field in `remote_write` config for Tempo. `headers`
   specifies HTTP headers to forward to the remote endpoint. (@alexbiehl)
 
+- [BUGFIX] Grafana Agent running as a Windows service should start automatically on startup
+  (@mattdurham) 
+
 # v0.13.1 (2021-04-09)
 
 - [BUGFIX] Validate that incoming scraped metrics do not have an empty label
```

Comment:
If we're cutting a release the bugfix from 0.13.1 should also be included in the changelog